frontend/src/components/ActionBar.tsx

```diff
@@ -1,15 +1,7 @@
-//import './ActionBar.css';  
-import { ReactElement } from 'react'
-type ActionBarProps = {
-  children: ReactElement
-}
+import { PropsWithChildren } from 'react'
 
-const ActionBar: React.SFC<ActionBarProps> = (props) => {
-  return (
-    <div className="mb-8 flex gap-2">
-        {props.children}
-    </div>    
-  )
+const ActionBar: React.FC<PropsWithChildren<{}>> = props => {
```

Comment:
E.g. of one of the "we don't need a type, just `PropsWithChildren<{}>`" -- maybe there's even a better macro, but this looks fine? Less lines